__tests__/api/app.test.js

```diff
@@ -12,35 +13,55 @@ beforeEach(() => {
   return seed(seedTestData);
 });
 
-describe('Testing app', () => {
-  describe('Endpoint /api/topics', () => {
-    describe('Method GET', () => {
-      test('GET: 200 status', () => {
-        return request(app).get('/api/topics').expect(200);
+describe("Testing app", () => {
+  describe("Endpoint /api/topics", () => {
+    describe("Method GET", () => {
+      test("GET: 200 status", () => {
+        return request(app).get("/api/topics").expect(200);
       });
 
-      test('GET: 200 status with all topics data and properties', () => {
+      test("GET: 200 status with all topics data and properties", () => {
         return request(app)
-          .get('/api/topics')
+          .get("/api/topics")
           .expect(200)
           .then(({ body }) => {
             const topics = body;
             expect(Array.isArray(topics)).toBe(true);
             expect(topics).toHaveLength(3);
 
             topics.forEach((item) => {
-              expect(item).toHaveProperty('slug', expect.any(String));
-              expect(item).toHaveProperty('description', expect.any(String));
+              expect(item).toHaveProperty("slug", expect.any(String));
+              expect(item).toHaveProperty("description", expect.any(String));
             });
           });
       });
 
-      test('GET: 404 status with msg Not Found when given a wrong endpoint', () => {
+      test("GET: 404 status with msg Not Found when given a wrong endpoint", () => {
         return request(app)
-          .get('/api/topicsss')
+          .get("/api/topicsss")
           .expect(404)
           .then(({ res }) => {
-            expect(res.statusMessage).toBe('Not Found');
+            expect(res.statusMessage).toBe("Not Found");
+          });
+      });
+    });
+  });
+
+  describe("Endpoint /api", () => {
+    describe("Method GET", () => {
+      test("GET: 200 status", () => {
+        return request(app).get("/api").expect(200);
+      });
+
+      test("GET: 200 responds with an object describing all the available endpoints on your API", () => {
+        return request(app)
+          .get("/api")
+          .expect(200)
```

Comment:
Here also, the line 59 `expect(200` is a repeated line because this is covered in the test before this one

Reply:
done